lstchain/spectra/crab.py

```diff
@@ -46,19 +46,19 @@ def crab_hegra(E):
 
     Returns
     -------
-    dFdE: `numpy.ndarray` differential energy spectrum. 
+    dFdE: `numpy.ndarray` differential energy spectrum.
           astropy.units.quantity.Quantity units: 1/u.TeV / u.cm**2 / u.s
     par: `dict` with spectral parameters
     """
 
-    f0    = 2.83e-11 / u.TeV / u.cm**2 / u.s
+    f0    = 2.83e-14 / u.GeV / u.cm**2 / u.s
     alpha = -2.62
-    e0    = 1 * u.TeV
+    e0    = 1000 * u.GeV
 
     par_var = [f0, alpha, e0]
     par_dic = ['f0', 'alpha', 'e0']
     par = dict(zip(par_dic, par_var))
 
     dFdE  = f0 * np.power(E / e0, alpha)
 
-    return dFdE.to(1/u.TeV / u.cm**2 / u.s), par
+    return dFdE.to(1/u.GeV / u.cm**2 / u.s), par
```

Comment:
There was a problem because in some places energies were in TeV (spectrum of crab, protons...) and in others (energies from dl2 files, limits of energy bins...), energies were in GeV. That was one of the main reasons from the scaling problems of the curves I showed you at first. I decided to put all energies of the code in GeV, and only transform to TeV at the time to plot the sensitivity curve.

Reply:
Long-standing issue, related to #132